features/support/paths.rb

```diff
@@ -262,6 +262,12 @@ def path_to(page_name)
     when /^the [mM]y account page$/
       '/my/account'
 
+    when /^the [aA]ccess [tT]oken page$/
+      '/my/access_token'
+
+    when /^the [sS]ettings page$/
```

Comment:
But this would only be effective if one were to try to visit the first administration settings tab without explicitly specifying the tab but rather rely on it being rendered by default when no tab is given?